node/src/reactor/joiner.rs

```diff
@@ -902,6 +917,12 @@ impl reactor::Reactor for Reactor {
                 req.answer(Err(Cow::Borrowed("node is joining, no running consensus")))
                     .ignore()
             }
+            #[allow(unused)]
+            JoinerEvent::ContractRuntimeRequest(req) => reactor::wrap_effects(
+                JoinerEvent::ContractRuntime,
+                self.contract_runtime
+                    .handle_event(effect_builder, rng, req.into()),
+            ),
```

Comment:
Isn't this a duplicate branch? We seem to be matching on `ContractRuntimeRequest` above already. Am I missing something?

Reply:
Yes, good catch. That's some debug leftover removed in https://github.com/casper-network/casper-node/pull/2614/commits/b984c5db0fe80ece4101e22614e54d40b036c62b